app/models/updateRegisteredDetails/UpdateContactDetails.scala

```diff
@@ -16,10 +16,20 @@
 
 package models.updateRegisteredDetails
 
+import models.Contact
 import play.api.libs.json._
 
 case class UpdateContactDetails (fullName: String, position: String, phoneNumber: String, email: String)
```

Comment:
I think ContactDetails would be a more suitable name for this model.

Reply:
Contact is what it is called across all the services it is part of the RetrievedSubscription model.

Reply:
Not sure why it tagged line 19. I was referring to UpdateContactDetails as an unsuitable name.